### DISCUSSION TITLE
Admin user page is broken in the latest release

### DISCUSSION THREAD

**Discussion by tapashdatta** [dani-garcia/vaultwarden#2885] [Q&A]
I had everything working fine. I have just updated to latest version and when I try to visit https://domain.com/admin/users/overview it throws an error

<img width="980" alt="Screenshot 2022-11-02 at 7 06 52 AM" src="https://user-images.githubusercontent.com/5366545/199374215-ff676c96-f8b5-4d2b-b375-9e23ba4d1dc9.png">

**Comment by BlackDex** [comment]
First go to /admin, login, and after that you can access it.

**Comment by bokkabonga** [comment]
I had this happening today to me as well. Was deleting some old users and while deleting one user i was redirected to this page. Not sure if maybe my session timed out, since i don´t know how long it takes for this to happen. But i was simply able to log back in after going to /admin again

**Reply by BlackDex** [reply]
@bokkabonga the timeout is 20min for the admin token.

**Reply by bokkabonga** [reply]
Checks out, then nothing to add here from my side
